app/inputs/collection_select_input.rb

```diff
@@ -116,7 +116,7 @@ def include_blank_as_string
 
   def prompt_as_string
     if options[:prompt] == true
-      I18n.translate('helpers.select.prompt', default: 'Please select')
+      I18n.t('helpers.select.prompt', default: 'Please select')
```

Comment:
minor, but adding `!` will raise on missing translations. I think this is our preferred behavior:
 
```suggestion
      I18n.t!('helpers.select.prompt', default: 'Please select')
```

Reply:
(aligning this entire app to that behavior might just be worth separate PRs)

Reply:
This was an automated fix from rubocop. I'm happy to split out the rubocop fixes into a separate PR as long as CI doesn't block on the linter.